bundle/jsky.app.ot/src/main/scala/jsky/app/ot/ags/BagsManager.scala

```diff
@@ -305,7 +347,7 @@ object BagsManager {
         _ <- logTransition(state, newState)
         _ <- sideEffect
       } yield ()
-
+      fireBagsStatusChanged(obs.getNodeKey, state.status, newState.status)
       action.unsafePerformIO()
```

Comment:
This may be a bit nitpicky but the status change hasn't actually happened until after the action is performed.  It might be best to move the event triggering after the action?